### DISCUSSION TITLE
Instantiating a model config for Azure OAI using AAD instead of `api_key`

### DISCUSSION THREAD

**Discussion by labeebee** [microsoft/promptflow#3552] [Q&A]
Hello,

I am trying to use Prompt Flow and my organization has enforced AAD for accessing Azure OAI resources. But I see no way to define a model config using `AzureOpenAIModelConfiguration()`. Can anyone help?

**Comment by brynn-code** [comment]
Hi, thanks for reaching us! I'll convert this discussion to a feature for better tracking :)

**Reply by brynn-code** [reply]
I'll engage related member, let's talk there for the follow up things.
